website/content/en/docs/building-operators/golang/migration.md

```diff
@@ -128,6 +149,45 @@ type Memcached struct {...}
 type MemcachedList struct {...}
 ```
 
+## Webhooks
+
+SDK version `1.0.0` and later has support for webhooks by the CLI. If your project doesn't require any webhooks, you can skip this section. However, if have been using it via customizations in your project, you should use the tool to re-scaffold the webhooks.  
+
+A webhook can only be scaffolded for a pre-existent API in your project. Then, for each case you will run the command `operator-sdk create webhook` providing the `--group`, `--kind` and `version` of the API based on the flags that need to be used.
+
+The valid flags for its types are: `--defaulting`, `--programmatic-validation` and `--conversion`. Use the same type that was used for scaffolding. To create `defaulting` and `validating` webhook:
```

Comment:
```suggestion
The valid flags for its types are: `--defaulting`, `--programmatic-validation` and `--conversion`. Use the same type that was used for scaffolding. To create `defaulting` and `programmatic-validation` webhook:
```

Reply:
`validating ` and `defaulting ` are the types. ``programmatic-validation` is the flag.